### ISSUE TITLE
Add FAQ to the footer

### ISSUE THREAD

**Issue by tiagocostasilva** [FightPandemics/FightPandemics#541]
Hi Team,
FAQ section will be needed on the footer for MVP and then later the design will be updated.

Already aligned with Yu, please insert FAQ on the footer at the left side separated by a "|" .

![image](https://user-images.githubusercontent.com/45389561/83569221-0d5baa80-a51c-11ea-89f0-8f1f323d8cf0.png)

**Important note:** Please validate it also on mobile to see how it fits the screen

Thanks,
Tiago

**Comment by PriyalPatney** [comment]
I'm working on this.

**Comment by stavares843** [comment]
seems this is already done, going to https://staging.fightpandemics.work/ already has the footer

**Comment by tiagocostasilva** [comment]
Not yet corrected @stavares843 , the FAQ is missing: 
![image](https://user-images.githubusercontent.com/45389561/83695905-ffc52400-a5f2-11ea-8e5d-a876b76715c3.png)

**Comment by stavares843** [comment]
ohh k

thanks

**Comment by PriyalPatney** [comment]
what should go inside the link?

**Comment by tiagocostasilva** [comment]
Waiting on an answer https://fightpandemics.slack.com/archives/C010QR5D4KY/p1591379359060100

**Comment by tiagocostasilva** [comment]
@PriyalPatney  can you check this one https://github.com/FightPandemics/FightPandemics/issues/616 ?

**Comment by PriyalPatney** [comment]
on it

**Comment by idjevm** [comment]
hey @PriyalPatney, I actually just noticed that you have to have your link set up this way -> `<Link to={"/faq"}>FAQ</Link>`, I can review once you open up a PR.

**Comment by PriyalPatney** [comment]
yes @idjevm , I'll request PR shortly!
